app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<nav class="navbar bg-primary d-flex justify-content-between p-1">
+  <%= link_to root_path do %>
+    <i class="bi bi-arrow-left btn"></i>
+  <%end%>
+  <h3>LOG IN</h3>
+  <h4>NEXT</h4>
+</nav>
+
+<div class="main-page">
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="inputs">
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class:"input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class:"input" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class:"btn" %>
+  </div>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Kindly implement the splash screen. When I load your app it takes me straight to the login page, but I should be taken to the splash screen instead. Kindly fix that.

  Your app | Splash screen design
  :---:|:---:
  ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/ef63692e-139c-4caa-9dbf-89353ec27bd7) | ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/557255a3-2ffa-4798-8726-3479ea030bea)